newIDE/app/src/EventsFunctionsExtensionsLoader/index.js

```diff
@@ -144,6 +145,29 @@ const loadProjectEventsFunctionsExtension = (
   });
 };
 
+/**
+ * Get the list of mandatory include files when using the
+ * extension.
+ */
+const getExtensionIncludeFiles = (
+  project: gdProject,
+  eventsFunctionsExtension: gdEventsFunctionsExtension,
+  options: Options,
+  codeNamespacePrefix: string
+): Array<string> => {
+  return mapFor(0, eventsFunctionsExtension.getEventsFunctionsCount(), i => {
+    const eventsFunction = eventsFunctionsExtension.getEventsFunctionAt(i);
+
+    const codeNamespace = getFreeFunctionCodeNamespace(
+      eventsFunction,
+      codeNamespacePrefix
+    );
+    const functionName = codeNamespace + '.func'; // TODO
```

Comment:
Could you explain what's the goal of the TODO?

Reply:
I'm not sure. Probably to do an helper function for it.

Reply:
This was an old todo from myself. The goal was to do an helper yes.